File: c++/a001~a020/a005.cpp
#include <iostream>
using namespace std;

int main(int argc, char const *argv[]){
    int a, b, c, d, range;
    for (cin >> range; range > 0; range--){
        cin >> a >> b >> c >> d;
        if (d - c == b - a){
            cout << a << " " << b << " " << c << " " << d << " " << (b - a + d) << '\n';
        } else {
            cout << a << " " << b << " " << c << " " << d << " " << (int)(b / a) * d << '\n';
        }
    }
    return 0;
}

File: python/a001~a020/a005.py
for i in range(int(input())):
    a, b, c, d = map(int, input().split())
    if (d - c == b - a):
        print(a, b, c, d, (b - a + d))
    else:
        print(a, b, c, d, b//a*d)
